### ISSUE TITLE
Time series background shading is incorrect

### ISSUE THREAD

**Issue by daviswx** [ncsco/pinemap-dss-help#103]
The time series seems to be using the default colors (green to blue) instead of the tool-specific palettes.  This is happening on both the PINEMAP DSS and Climate Voyager.

**Comment by daviswx** [comment]
The colors seem to be working again on both dev sites.  The only change I made was moving the code that applies the color gradient down just a bit.  Theoretically, the placement shouldn't make a difference, but apparently it does.

I'm not exactly sure what suddenly caused this to break, but it wasn't a recent change I made because even on the alpha and beta versions of the DSS, the time series colors aren't showing up correctly.  I did notice that Highcharts recently updated to a new version, and while I didn't see anything in the release notes that specifically addressed the function I'm using, I assume that something minor in the new version must have caused it.

That does make me a bit worried that other time series functionality may not be working as expected, but so far, I haven't found anything that seems wrong.
